zsh/PKGBUILD

```diff
@@ -43,6 +45,7 @@ prepare() {
   patch -p1 -i "${srcdir}/msysize.patch"
   patch -p1 -i "${srcdir}/zsh-5.0.6-1.patch"
   patch -p1 -i "${srcdir}/msys-symlink-hack.patch"
+  patch -p1 -i "${srcdir}/fix-zpty-module.patch"
```

Comment:
Can you please add a comment above this with a link to the upstream issue. So we know when we can drop the patch.

Reply:
ok, but `configure.ac` change is needed for cygwin/msys2 to use ptmx device, if zsh upstream not fix.